api/controllers/subscription.js

```diff
@@ -199,27 +208,32 @@ async function syncSubscriptions(req, res) {
   }
 }
 
-function mapRemoteSubscr(subscription) {
+function mapRemoteSubscr(subscription, paypalPlans) {
   const subscr = {
     subscriptionId: subscription.productId,
     name: subscription.listings[0].title,
     status: 'active',
     platform: 'android-playstore',
```

Comment:
now the platform is also Web

Reply:
The source of information to syncronize the subscription and plans is always the play store, then we link those subscriptions to the other platforms like Paypal